doc/release-notes/8947-payara-update.md

```diff
@@ -0,0 +1,76 @@
+# Update to Payara 5.2022.3 highly recommended
+
+*NOTE: this might be rephrased to "required" depending on https://github.com/IQSS/dataverse/pull/8915 being merged.*
+
+With lots of bug and security fixes included, we encourage everyone to update to Payara 5.2022.3 as soon as possible.
+
+**Note:** with the approaching EOL for the Payara 5 Community release train it's likely we will switch to a
+yet-to-be-released Payara 6 in the not-so-far-away future.
+
+We recommend you ensure you followed all update instructions from the past releases regarding Payara.
+(latest Payara update was for [v5.6](https://github.com/IQSS/dataverse/release/v5.6))
```

Comment:
Should the GitHub url here be https://github.com/IQSS/dataverse/releases/tag/v5.6? (If I try what is here "release/v5.6" I get "not found")

Reply:
Good catch - will add fix.